.eslintrc

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "rules": {
+    "quotes": ["error", "single"],
+    "semi": ["error", "always"],
+
+    "comma-spacing": ["error", { "before": false, "after": true }],
+
+    /*    "space-in-brackets": ["error", "always", {
+          "singleValue": false,
+          "objectsInArrays": false,
+          "arraysInArrays": false,
+          "arraysInObjects": false,
+          "objectsInObjects": false,
+          "propertyName": true
+        }],*/
```

Comment:
Remove commented out stuff and while you are doing that rename whole file to `.eslintrc.json`...